packages/math-input/src/components/keypad/operators-page/pre-algebra-buttons.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import Keys from "../../../data/key-configs";
+import {SecondaryKeypadButton} from "../keypad-page-items";
+
+type Props = {
+    onClickKey: (keyConfig: string) => void;
```

Comment:
Someday, I'm hoping that these keyconfig keys will have their own type. 

No action required.